README.md

```diff
@@ -35,9 +35,6 @@ Here are the basic requirements to run the project:
 
 NODE_PATH=src
 
-REACT_APP_ASSETS_URL=https://builder-packs-prod.now.sh
-REACT_APP_CONTENT_SERVER_URL=https://content.decentraland.org
```

Comment:
Would you mind removing this environment variable from the Vercel configuration too? (the `now.json` file)

Reply:
Done!